.scripts/run_config.sh

```diff
@@ -37,16 +37,25 @@ cleanup() {
 }
 trap cleanup EXIT
 
+mkdir -p "${root_path}/tmp"
+
 if [ $target == "Default" ]; then
 
-  PASSWORD=${DB_PASSWORD:-$CREDENTIALS}
+  storage_target=$(basename $(dirname "${model_path}"))
```

Comment:
This is a good argument for the config supporting "Redshift" and "Snowflake" instead of/as well as "Default"